doctr/io/elements.py

```
@@ -253,6 +258,89 @@ def synthesize(self, **kwargs) -> np.ndarray:
+            block_div = SubElement(body, 'div', attrib={
+                'class': 'ocr_carea',
+                'id': f'block_{block_count}',
+                'title': f'bbox {int(round(xmin * width))} {int(round(ymin * height))} \
+                    {int(round(xmax * width))} {int(round(ymax * height))}'
+            })
+            paragraph = SubElement(block_div, 'p', attrib={
+                'class': 'ocr_par',
+                'id': f'par_{block_count}',
+                'title': f'bbox {int(round(xmin * width))} {int(round(ymin * height))} \
+                    {int(round(xmax * width))} {int(round(ymax * height))}'
+            })
+            block_count += 1
+            for line in block.lines:
+                (xmin, ymin), (xmax, ymax) = line.geometry  # type: ignore[misc]
+                # NOTE: baseline, x_size, x_descenders, x_ascenders is currently initalized to 0
+                line_span = SubElement(paragraph, 'span', attrib={
+                    'class': 'ocr_line',
+                    'id': f'line_{line_count}',
+                    'title': f'bbox {int(round(xmin * width))} {int(round(ymin * height))} \
+                        {int(round(xmax * width))} {int(round(ymax * height))}; \
+                        baseline 0 0; x_size 0; x_descenders 0; x_ascenders 0'
+                })
+                line_count += 1
+                for word in line.words:
+                    (xmin, ymin), (xmax, ymax) = word.geometry  # type: ignore[misc]
+                    conf = word.confidence
+                    # NOTE: rotated boxes are currently not supported in the XML export and can be skipped
+                    if (ymax - ymin) > (xmax - xmin) and not skip_rotated_boxes:
+                        raise ValueError(
+                            "Rotated boxes are currently not supported in XML export")
+                    elif (ymax - ymin) > (xmax - xmin) and skip_rotated_boxes:
+                        continue
```

cf my comment earlier, let's raise an error if there are rotated bbox, and proceed otherwise